plugins/modules/gitlab_group_variable.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
The module must be in the appropriate subdirectory (check where the other gitlab modules are), and there must be a symlink from this place to the real module file.

Reply:
Thanks, fixed in https://github.com/ansible-collections/community.general/pull/786/commits/6a094e2b8f90c7b68309bb6989fce79a355a3b70